### ISSUE TITLE
Issues when installing openSUSE and SLES

### ISSUE THREAD

**Issue by GabyCT** [kata-containers/tests#1243]
@marcov when I am trying to install packages on openSUSE and SLES using Azure images, I am getting the following error
```
./setup.sh
Set up environment
Install chronic
Loading repository data...
Warning: Repository 'openSUSE-Leap-42.3-Update-Non-Oss' appears to be outdated. Consider using a different mirror or server.
Reading installed packages...
'moreutils' is already installed.
No update candidate for 'moreutils-0.61-1.1.x86_64'. The highest available version is already installed.
Resolving package dependencies...

Nothing to do.
Install curl
Can't locate IPC/Run.pm in @INC (you may need to install the IPC::Run module) (@INC contains: /usr/lib/perl5/site_perl/5.18.2/x86_64-linux-thread-multi /usr/lib/perl5/site_perl/5.18.2 /usr/lib/perl5/vendor_perl/5.18.2/x86_64-linux-thread-multi /usr/lib/perl5/vendor_perl/5.18.2 /usr/lib/perl5/5.18.2/x86_64-linux-thread-multi /usr/lib/perl5/5.18.2 /usr/lib/perl5/site_perl .) at /usr/bin/chronic line 54.
BEGIN failed--compilation aborted at /usr/bin/chronic line 54.
```
Any idea? thanks

**Comment by marcov** [comment]
Hi @GabyCT, can you give some more details of what you are doing?

The error is coming when launching `chronic`, that is a Perl application and it is because the `IPC/Run` lib is not found.

Indeed, I can reproduce the issue in a openSUSE 42.3 container.  Checking the `moreutils` package dependencies, IPC/Run is not there:
```
# rpm -q --requires moreutils
/usr/bin/perl
libc.so.6()(64bit)
libc.so.6(GLIBC_2.2.5)(64bit)
libc.so.6(GLIBC_2.3)(64bit)
libc.so.6(GLIBC_2.3.4)(64bit)
libc.so.6(GLIBC_2.4)(64bit)
libc.so.6(GLIBC_2.7)(64bit)
perl
perl-Time-Duration
perl-TimeDate
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsLzma) <= 4.4.6-1
```
This looks like a bug in the `chronic` package to me, resolved since Leap 15.0. 
A workaround is to install the missing package manually, that is called `perl-IPC-Run`.

However, wouldn't it be possible to use a Leap 15.0 image instead?

**Comment by GabyCT** [comment]
@marcov , currently I change to openSUSE Leap 15 and it is working the installation script. I have only issues while running the shimv2 tests https://github.com/kata-containers/tests/issues/1251.
